src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => (
```

Comment:
The Navigation component should be exported as a default export. Change it to 'export default const Navigation = () => (...)'.

Reply:
I don't understand why I need to do this.And I get errors with this syntax - 'export default const Navigation = () => (...)'.